src/main/java/entities/Deck.java

```diff
@@ -3,14 +3,20 @@
 import java.util.Collections;
 import java.util.List;
 
-
+/**
+ * A standard 52-card playing deck containing 52 Card objects, each representing 1 of the 52 unique cards in a
+ * standard deck.
+ */
 public class Deck {
-    // Constants and instance attributes. RANKS is an array of all possible ranks, SUITS is an array of all possible suits
+    // Constants and instance attributes. RANKS is an array of all possible ranks,
+    // SUITS is an array of all possible suits
     private List<Card> cards = new ArrayList<>();
-    private static final String[] RANKS = {"A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"};
-    private static final char[] SUITS = {'H', 'S', 'D', 'C'};
+    private final String[] RANKS = {"A", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K"};
+    private final char[] SUITS = {'H', 'S', 'D', 'C'};
```

Comment:
`RANKS` and `SUITS` can remain static, will just have to change the constructor loop to `Deck.RANKS`